server/rootless.go

```diff
@@ -45,3 +83,27 @@ func makeOCIConfigurationRootless(g *generate.Generator) {
 
 	g.SetLinuxCgroupsPath("")
 }
+
+// getAvailableV2Controllers returns the entries in /sys/fs/cgroup/<SELF>/cgroup.controllers
```

Comment:
can we memoize the return value with `sync.Once`?

Reply:
I don't think so, the file content can change dynamically

Reply:
but something that can be fixed by restarting CRI-O instead of having the cost for each container?

Anyway, not a blocker, we can deal with it later if we need to

Reply:
or, we could asynchronously check if the file has changed, and update if so. that may cause a delay in successful container creation if it does change, but the normal case would not add overhead per container create

Reply:
This is not a file but two (`/proc/self/cgroup` and `/sys/fs/cgroup/<SELF>/cgroup.controllers`), and these are not usual files, but rather an API to the kernel. For some files, it is possible to register an event handler to be notified by the kernel if the file is being changed, but I am not sure if it's possible for these two.

Even if it is, I don't think it makes sense to do so (add a watcher, update the contents in async manner), as there is not too much contents to read/parse, and it would be a gross complication with little performance benefit (shave off a few ms on a rootless container start).